compliance_checker/acdd.py

```diff
@@ -69,7 +69,7 @@ def check_recommended(self, ds):
     def check_suggested(self, ds):
         return [
             'contributor_name',
-            ('contributor_role', ['principalInvestigator', 'author']),
+            'contributor_role',
```

Comment:
This former syntax was to say that not only must `contributer_role` need to exist, it needed to be one of those two values.  I got this from some document at one point as it was the requirements. Has it changed?